src/Avalonia.Controls/ContextMenu.cs

```diff
@@ -47,23 +71,60 @@ public ContextMenu(IMenuInteractionHandler interactionHandler)
         /// </summary>
         static ContextMenu()
         {
-            ItemsPanelProperty.OverrideDefaultValue(typeof(ContextMenu), DefaultPanel);
+            ItemsPanelProperty.OverrideDefaultValue<ContextMenu>(DefaultPanel);
+            PlacementModeProperty.OverrideDefaultValue<ContextMenu>(PlacementMode.Pointer);
             ContextMenuProperty.Changed.Subscribe(ContextMenuChanged);
         }
 
+        /// <summary>
+        /// Gets or sets the placement mode of the popup in relation to the <see cref="PlacementTarget"/>.
```

Comment:
```suggestion
        /// Gets or sets the placement mode of the context menu in relation to the <see cref="PlacementTarget"/>.
```

Wouldn't this be clearer?